### PR TITLE
animation on load for the images is added

### PR DESCRIPTION
**Pull request by arslanbekzhaparov**

### CODE REVIEW

File: src/components/Card/index.tsx [hunk 1]
```diff
@@ -17,6 +32,8 @@ const ProjectImageContainer = styled.div`
 	perspective: 1px;
 	filter: drop-shadow(0px 4px 30px rgba(0, 0, 0, 0.3));
 	overflow: visable;
+
+	animation: ${fadeIn} 3s linear 0s;
```
Comment (colesperks) [inline src/components/Card/index.tsx:36]: 3s feels a bit long. Lets Shorten this to 1s

Comment (arslanbekzhaparov) [reply src/components/Card/index.tsx:36]: yep was also thinking about that)

**Review by colesperks — APPROVED** [review]
